README.md

```diff
@@ -1,30 +1,44 @@
-# SourceExplorer
-Last updated by | Egor Belekhov | Nov 23, 2021 at 11:23 AM GMT+3
+# GitRunner
+Last updated by | Egor Belekhov | Nov 24, 2021 at 10:03 AM GMT+3
 ***
 
-## Overview
+# GitRunner
```

Comment:
I don't think we need this header for a second time.

Reply:
maybe I misunderstood about the need for a header. @alsenichev do I need it?

Reply:
removed